trie-db/src/lookup.rs

```diff
@@ -211,7 +229,13 @@ where
 				full_key,
 				cache,
 				|value, _, full_key, _, _, recorder| match value {
-					ValueOwned::Inline(value, hash) => Ok((hash, Some(value.clone()))),
+					ValueOwned::Inline(value, hash) => {
```

Comment:
likewise, should we record TrieAccess::InlineHash? Probably not, but maybe a comment.

Reply:
function `look_up_with_cache` is recording Value access when non inline, it can probably also do for the inline case.

Reply:
> function look_up_with_cache is recording Value access when non inline, it can probably also do for the inline case.

Not really sure what I meant here at the time, make only sense if there was node value in inline node (which is doable but not really a sensible configuration).